tendermint/src/serializers/custom.rs

```diff
@@ -47,7 +47,7 @@ where
                     None
                 } else {
                     Some(block::parts::Header {
-                        total: tmp_id.parts.total,
+                        total: tmp_id.parts.total as u32, // Todo: manage overflow
```

Comment:
Just making sure we do not forget about this TODO.

Reply:
Covered in the next commit.